lib/apogeeDetection/apogeeDetection.cpp

```diff
@@ -57,5 +65,16 @@ bool ApogeeDetection::atApogee(double altitude, double accel_z) {
 	return false;
 }
 
+void ApogeeDetection::updateAlt(float* data) {
+	altitude = data[0];
+}
+
+void ApogeeDetection::updateAcc(float* data) {
+	acc_z = data[2];
+}
```

Comment:
nice.
send in just a single float though. 
it should be called like `updateAlt(farrbconvert.sensorReadings[0])`